api/tests/routes/backoffice/custom_reimbursement_rules_test.py

```diff
@@ -121,6 +121,30 @@ def test_list_rules_by_offer_name(self, authenticated_client):
 
         assert html_parser.extract_pagination_info(response.data) == (1, 1, len(rows))
 
+    def test_list_rules_by_category(self, authenticated_client):
+        offer_rule_music = finance_factories.CustomReimbursementRuleFactory(subcategories=["FESTIVAL_MUSIQUE"])
+        finance_factories.CustomReimbursementRuleFactory(subcategories=["CARTE_MUSEE"])
+
+        with assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, categories=["MUSIQUE_LIVE"]))
+
+        assert response.status_code == 200
```

Comment:
Je suis déjà passé sur tous les tests BO donc dans l'idéal... Peux-tu remettre la vérification du `status_code` dans le bloc `with`, afin de détecter une erreur de retour (par exemple une exception, la vraie erreur) avant de compter les requêtes ? Pareil en-dessous.
```suggestion
        with assert_num_queries(self.expected_num_queries):
            response = authenticated_client.get(url_for(self.endpoint, categories=["MUSIQUE_LIVE"]))
            assert response.status_code == 200

```